Desktop/widgets/listmodel.h

```diff
@@ -76,7 +76,8 @@ class ListModel : public QAbstractTableModel, public VariableInfoConsumer
 			void					setColumnsUsedForLabels(const QStringList& columns)						{ _columnsUsedForLabels = columns; }
 			void					setRowComponent(QQmlComponent* rowComponents);
 	virtual void					setUpRowControls();
-	const rowControlMap	&			getRowControls() const { return _rowControlsMap; }
+	const rowControlMap	&			getAllRowControls()											const		{ return _rowControlsMap;				}
+	RowControls*					getRowControls(const QString& key)							const		{ return _rowControlsMap.value(key);	}
```

Comment:
Shouldnt you check whether it is there first? This might add dummy shit

Reply:
QMap value does not add dummy shit!